.github/workflows/ios.yml

```diff
@@ -61,4 +61,4 @@ jobs:
             file_to_build="$(ls -A | grep -i \\.xcodeproj\$)"
           fi
           file_to_build=$(echo $file_to_build | awk '{$1=$1;print}')
-          xcodebuild test -scheme "$scheme" -"$filetype_parameter" "$file_to_build" -destination "platform=iOS Simulator,name=iPhone 15,OS=latest" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
+          xcodebuild test -scheme "$scheme" -"$filetype_parameter" "$file_to_build" -destination "platform=iOS Simulator,name=iPhone 15,OS=17.2" CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO
```

Comment:
fixes the CI job error:
```terminal
xcodebuild: error: Unable to find a device matching the provided destination specifier:
		{ platform:iOS Simulator, OS:latest, name:iPhone 15 }

	The requested device could not be found because no available devices matched the request.
```